src/Core/src/Platform/Windows/TextBlockExtensions.cs

```diff
@@ -125,5 +127,33 @@ internal static void UpdateTextPlainText(this TextBlock platformControl, IText l
 		{
 			platformControl.Text = label.Text;
 		}
+		
+		static XElement? ParseXhtml(string? html)
```

Comment:
Maybe we can move this into `LabelHtmlHelper` so that is is more easily testable.

Reply:
Mnn, then we should move the UpdateTextHtml extension method too, right?. But, the TextBlockExtensions class is also public and contains extension methods easily testeables.